### ISSUE TITLE
Catbots stuck on WAITING (both catbot-setup-cli and nullworks catbot-setup)

### ISSUE THREAD

**Issue by rosne-gamingyt** [explowz/cathook#41]
Same thing as last time, i tested without the auto vote update's changes and it works fine. Please revert [this](https://github.com/explowz/cathook/commit/804929e7908bed3e0f501e4c1314d1fc510160b7).

**Comment by snipergaming888** [comment]
yeah. the version with auto-vote fix works fine. i don't use the most recent.
